content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+able to pass your experiences and your knowledge to other people. Helping people grow and succeed is very important. Not only you're 
+actively using your time for others, but also you're increasing the area of impact. We also believe, that during mentoring, you can learn 
+a tremendous amount of things about yourself, efficient communication, setting goals, etc.
+
+Examples, of how to influence others:
+* Participate in or start a mentoring program at your company.
+* Actively demonstrate company values and culture through your actions
+* Review code, documents, proposals, etc.
+* Give feedback!
+
+## Communicate clearly and bring clarity
+![](./communication.png)
+
+Miscommunication is often a source of most problems at work (and in life). Misunderstanding can result in poor performance, subpar
+quality, stressed people, or wasted time. The role of staff engineers is to bring 
+[clarity and alignment](https://alexewerlof.substack.com/p/6-ways-staffs-reach-clarity).
+
+There are several things staff engineers should do to improve communication in their teams and company:
+* Listen carefully to others and understand their problems and point of view.
+* Make sure the team reaches a collective understanding.
+* [Start with why](https://simonsinek.com/books/start-with-why/) and explain why we work on certain things.
+* Lead by example and improve their communication skills. Show more junior engineers this is often a key to success.
+
+## Bring calm
+![](./calm.jpg)
+
+There are going to be some stressful situations at work. It can be either a tight deadline or an application incident. Staff engineers should
+show their experience and stay calm during the crisis. They should take a lead, communicate clearly and help with prioritization. Their role
+is to help team members do their work better.
+
+Another way to bring calm to the organization is by preparing for different failure scenarios much earlier. Staff engineers should assess
+possible risks even before the project starts. Then, during the ideation or implementation phase, they should systematically review the risks,
+think if mitigation plans still work, and adjust. The best way to get out of the crisis is to foresee it and prevent it from happening.
+
+So what to do to bring calm to the team?
+* Do a risk assessment, and pre-mortems and anticipate failure scenarios before they happen.
+* In critical situations take a lead and help the team solve the problem.
+* And again, communicate clearly and bring clarity.
+
+## Build trust and relationships
+![](./relationship.png)
+
+Last but not least, staff engineers are expected to build and cherish relationships at work. They should have relationships not only with 
+their closest team, but different stakeholders and other colleagues in the organization. They should listen carefully and understand the 
+needs of people and teams. This can help them better identify problems and empower others.
```

Comment:
`This can help them better identify problems and empower others.` this also sounds like a hanging sentence. I don't feel convinced by this with just the context of the entire paragraph.